report-viewer/src/libs/highlightjs/styles/edited/default.min.css

```diff
@@ -0,0 +1,82 @@
+/*!
+  Theme: Default
+  Description: Original highlight.js style
+  Author: (c) Ivan Sagalaev <maniac@softwaremaniacs.org>
+  Maintainer: @highlightjs/core-team
+  Website: https://highlightjs.org/
+  License: see project LICENSE
```

Comment:
If we embed others people code we shall not violate their license. Also: why is a copy of the CSS file required here?

Reply:
I have marked the attributes which I have edited in the local copy in order to keep my styling of code while maintaining the highlighted words. If I use highlight.js styles directly from the import I cannot override css and the the lines of code have large font, margins, paddings, don't use my font and have their own background which prevents the match colors from displaying properly.

Reply:
Maybe I'm wrong but you can always extend and overwrite CSS, e.g. by defining own styles marked with `!important`.

Reply:
See: https://www.w3schools.com/css/css_important.asp